go_logging_server/api/api.go

```
@@ -0,0 +1,42 @@
+
+// Starts up the api server
+func StartUpServer() {
+
+	/*Setup router*/
+	router := gin.Default()
+
+	//Initial endpoint setup
+	setupEndPoints(router)
+
+	/*Run router*/
+	router.Run("0.0.0.0:8090")
```

golangci-lint (1.64.8)

Error return value of `router.Run` is not checked

(errcheck)